internal/uvm/cpugroups.go

```diff
@@ -0,0 +1,289 @@
+package uvm
```

Comment:
Can you write a comment detailing what cpu groups are, their uses etc. in this file?

Reply:
I suppose I can, but they're publicly documented on MSDN.

Reply:
Then a link to that here would be nice 😃. Just a brief overview